Example/Example/AppDelegate.swift

```diff
@@ -0,0 +1,22 @@
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+    var appFlowController: AppFlowController!
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+
+        if let window = window {
+            appFlowController = AppFlowController(window: window)
+            appFlowController.start()
+        }
+
+        return true
```

Comment:
returning `true` here seems a bit lucid. What's the meaning of it? unless it can return false in some cases it can just return void imo

Reply:
This is an UIKit thing.


From the documentation:
`return false if the app cannot handle the URL resource or continue a user activity, otherwise return true. The return value is ignored if the app is launched as a result of a remote notification.`

Reply:
👍